components/Message/index.tsx

```diff
@@ -12,14 +12,23 @@ import {
 
 import {useHistory} from 'react-router-native'
 
-import AppHeaderLarge from '../AppHeaderLarge/index.js'
-import LoadingOverlay from '../LoadingOverlay/index.js'
+import AppHeaderLarge from '../AppHeaderLarge/index'
+import LoadingOverlay from '../LoadingOverlay/index'
 
 import Images from '../../assets/images'
 import AppStyles from '../../assets/styles'
 import Styles from './styles'
 
-function Message(props) {
+interface IMessage {
+  bgColor: string
+  title: string
+  textLight: String
+  text: string
+  path: string
+  children: JSX.Element
```

Comment:
Should children be an array of JSX.Elements? I know in the Setup Wizard we're accessing children at specific indexes so it seems somewhat likely that it'd be an array?